techniques/general/G56.html

```diff
@@ -52,111 +49,105 @@
             </ul>
          
       </section>
       <section class="example" id="example-2-sufficient-audio-contrast-between-a-narrator-and-background-music">
-         <h3>Example 2: Sufficient audio contrast between a narrator and background music</h3>
+         <h3>事例 2: ナレーターの音声と背景音楽との十分な音声コントラスト</h3>
          
-            <p>This example demonstrates a voice with music in the background in which the voice is the appropriate 20 DB above the background. The voice (foreground) is recorded at -17.52 decibels (average RMS) and the music (background) is at -37.52 decibels, which makes the foreground 20 decibels louder than the background.</p>
+            <p>この事例では、ナレーターの音声と背景音楽があり、主音声は背景音楽よりもちょうど 20 デシベル大きい。主音声 (前景) は -17.52 デシベル (二乗平均平方根)、音楽 (背景) は -37.52 デシベルで録音しているため、前景は背景よりも 20 デシベル大きいことになる。</p>
             <section id="audio-example">
-              <h4>Audio example</h4>
+              <h4>音声サンプル</h4>
               <p class="working-example">
-                  <a href="../../working-examples/audio-contrast/good-contrast.mp3">Audio Example: Foreground is 20 decibels above the background (mp3)</a>
+                  <a href="https://www.w3.org/WAI/WCAG22/working-examples/audio-contrast/good-contrast.mp3">音声サンプル: 前景音声が背景音楽よりも 20 デシベル大きい (mp3 形式)</a>
                 </p>
             </section>
             <section id="transcript-of-audio-example-good-contrast">
-              <h4>Transcript of audio example (good contrast)</h4>
+              <h4>音声サンプル (十分なコントラスト) のトランスクリプト</h4>
               <p>"Usually the foreground refers to a voice that is speaking and should be understood. My speaking voice right now is 20 decibels above the background which is the music. This is an example of how it should be done.."</p>
             </section>
             <section id="visual-example-of-the-recording-above">
-              <h4>Visual example of the recording above</h4>
-              <p>The audio example above is visually represented below in a snapshot of the file in an audio editor. A section is highlighted that contains foreground and  background. It is a much larger wave than the section that contains only background.</p>
+              <h4>音声サンプルを視覚的に示したもの</h4>
+              <p>音声サンプルを、音声エディタ上のファイルのスナップショットという形で、以下に視覚的に示してある。前景音声と背景音楽を含んだセクションは強調表示にしてある。背景音楽のみのセクションに比べて、波形がはるかに大きい。</p>
               <figure id="figure-visual-color-contrast">
-              	<img alt="A display showing a sound wave. The left third of the sound wave is just background noise, registering from -10 to +10 decibels. The remaining right-hand section is foreground and background noise showing a range from -50 to +40 decibels." src="img/Audio_3.jpg">
-              	<figcaption>Visual representation of sufficient contrast.</figcaption>
+              	<img alt="音波を表示するディスプレイ。音波の左側 3 分の 1 は背景ノイズで、-10 デシベルから +10 デシベルの範囲で表示される。残りの右側の部分は、前景ノイズと背景ノイズで、-50 デシベルから +40 デシベルの範囲で表示される。" src="img/Audio_3.jpg">
```

Comment:
forground (前景) はノイズではないので、「前景音」にするのはいかがでしょうか。
```suggestion
              	<img alt="音波を表示するディスプレイ。音波の左側 3 分の 1 は背景ノイズで、-10 デシベルから +10 デシベルの範囲で表示される。残りの右側の部分は、前景音と背景ノイズで、-50 デシベルから +40 デシベルの範囲で表示される。" src="img/Audio_3.jpg">
```